src/constants/Configurables.ts

```diff
@@ -0,0 +1,28 @@
+export default {
```

Comment:
You should maybe declare some interfaces for these to use them where you need them. The interface should have `name`, `options`and `unit`